### PR TITLE
WEBAPP-245: Add flow to categories module

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://integreat.atlassian.net/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: src/routes/categories/components/CategoryListItem.js [hunk 1]
```diff
@@ -4,11 +4,58 @@ import * as React from 'react'
 
 import CategoryModel from 'modules/endpoint/models/CategoryModel'
 import iconPlaceholder from '../assets/IconPlaceholder.svg'
-import {
-  CategoryCaption,
-  CategoryThumbnail, Row,
-  StyledLink, SubCategoryCaption, SubCategory
-} from './CategoryListItem.styles'
```
Comment (Schedulaar) [inline src/routes/categories/components/CategoryListItem.js]: Why did you move these out of the .styles folder?

File: src/routes/categories/components/CategoryTimeStamp.js [hunk 2]
```diff
@@ -3,8 +3,15 @@
 import React from 'react'
 import type { TFunction } from 'react-i18next'
 import { translate } from 'react-i18next'
-import { TimeStamp } from './CategoryTimeStamp.styles'
```
Comment (Schedulaar) [inline src/routes/categories/components/CategoryTimeStamp.js]: Same here

File: src/routes/categories/components/CategoryListItem.js (hunk 1 above)
Comment (klinzo) [reply src/routes/categories/components/CategoryListItem.js]: I think we agreed on doing this, see https://issues.integreat-app.de/browse/WEBAPP-314

Comment (Schedulaar) [reply src/routes/categories/components/CategoryListItem.js]: Hmm never seen this task before :D

But it should also be made within this task then  🙃

Comment (klinzo) [reply src/routes/categories/components/CategoryListItem.js]: Hmm I don't see a problem to do this here, makes the other task way smaller.

Comment (Schedulaar) [reply src/routes/categories/components/CategoryListItem.js]: Not that you need to revert it, but we shouldn't randomly start doing parts of a some different task :)
Also I'm not yet convinced, that we should even do it ^^
